.github/workflows/check.yaml

```diff
@@ -27,15 +34,12 @@ jobs:
   func:
     uses: canonical/bootstack-actions/.github/workflows/func.yaml@v3
     needs: lint-unit
-    strategy:
-      fail-fast: false
-      matrix:
-        command: ["TEST_JUJU3=1 make functional"]  # TEST_JUJU3 needed due https://github.com/openstack-charmers/zaza/blob/b22c2eed4c322f1dfc14ffb2d31e0dd18c911a40/setup.py#L47 to support Juju3+
-        juju-channel: ["3.4/stable"]
-        
     with:
-      command: ${{ matrix.command }}
-      juju-channel: ${{ matrix.juju-channel }}
+      # TEST_JUJU3 needed due https://github.com/openstack-charmers/zaza/blob/b22c2eed4c322f1dfc14ffb2d31e0dd18c911a40/setup.py#L47 to support Juju3+
+      # Don't tear down the OpenStack model if tests fail and --keep-faulty-model is
+      # specified - this allows for interactive debugging of failed runs
```

Comment:
Could you explain for future travellers what you mean by "allows for interactive debugging"?  It's not clear unless you understand our internal infrastructure and permissions - ie. this is only applicable to our team internally, and not everyone may know how to do this.